multiplied

```diff
@@ -0,0 +1 @@
+src/Common/GfMul.c:253:/* element in GHASH is multiplied by x^8, this function will return  */
```

Comment:
This file is some artifact of your typo search process, I presume - please remove it from the PR.